apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.html

```diff
@@ -0,0 +1,149 @@
+                                     <ng-container matColumnDef="tax_percentage">
+                                        <th mat-header-cell *matHeaderCellDef>
+                                            {{ localeData?.tax_percentage }} (%)
+                                        </th>
+                                        <td mat-cell *matCellDef="let element" width="18%">
+                                            {{ element?.taxPercentage }}
+                                        </td>
+                                        <td class="bg-gray fw-bold" mat-footer-cell *matFooterCellDef></td>
+                                    </ng-container>
+        
+                                     <!-- Tax Collected -->
+                                     <ng-container matColumnDef="tax_collected">
+                                        <th mat-header-cell *matHeaderCellDef>
+                                            {{ localeData?.tax_collected }}
+                                        </th>
+                                        <td mat-cell *matCellDef="let element" width="18%">
+                                            {{ element?.taxTotal }}
+                                        </td>
+                                        <td class="bg-gray fw-bold" mat-footer-cell *matFooterCellDef>
+                                            <amount-field
+                                                class="d-inline-block"
+                                                [amount]="(taxWiseReport$ | async)?.totalTaxCollected"
+                                                [currencySymbol]="activeCompany?.baseCurrencySymbol"
+                                                [currencyCode]="false"
+                                            ></amount-field>
+                                        </td>
+                                    </ng-container>
+        
+                                    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+                                    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+                                    <tr mat-footer-row *matFooterRowDef="displayedColumns"></tr>
+                                </table>
+                            </div>
+                        </div>
+
+                        <!-- No Data Found Message -->
+                        <div class="no-data" *ngIf="(taxWiseReport$ | async)?.responses?.length === 0 && !(isLoading$ | async) && !isTaxApiInProgress">
+                            <h1 *ngIf="!getFormControl('taxNumber')?.value">{{ localeData?.no_sales_tax_available }}</h1>
+                            <h1 *ngIf="getFormControl('taxNumber')?.value">{{ commonLocaleData?.app_no_result_found }}</h1>
+                        </div>
+
+                        <!-- Pagination  -->
+                        <div class="pagination-wrapper justify-content-end" *ngIf="pagination?.totalPages > 1">
```

Comment:
Make changes according to the API count. When it is more than the API items response, it will show.